recipes/aaf/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AafConan(ConanFile):
+    name = "aaf"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sourceforge.net/projects/aaf/"
+    description = "A  cross-platform SDK for AAF. AAF is a metadata management system and file format for use in professional multimedia creation and authoring."
+    topics = ("aaf", "multimedia", "crossplatform")
+    license = "AAF SDK PUBLIC SOURCE LICENSE"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+            "shared": [True, False],
+            "fPIC": [True, False]
+            }
+    default_options = {
+            "shared": False,
+            "fPIC": True
+            }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio" and self.settings.compiler.runtime in ("MT", "MTd"):
```

Comment:
Can you explain why it's causing problems?
Usually, static runtime only causes problems when building shared libraries.
But static libraries with static runtime are usually fine.

Reply:
I just didn't manage to make the CI green in this case. I should add a TODO FIXME comment then.